CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+- We also expect group/topic leaders/coordinators/facilitators to provide a quick (daily if possible) summary via the wiki so everybody can get an idea on it.
+
+## How can I participate at this event?
+
+Join the [Google group mailing list](https://groups.google.com/forum/#!forum/virtual-biohackathon) and the [virtualbiohackathoncovid19 slack](https://join.slack.com/t/virtualbiohac-xt62674/shared_invite/zt-cuur40oj-wdrVz50NocwVrH7vgKTdPg) and join the conversation! No approval needed. 
+
+## How can join the hackthon?
+- Please have a look at the [GitHub wiki](https://github.com/virtual-biohackathons/covid-19-bh20/wiki) to see the existing projects. 
+- When you’ve found a project you’d like to contribute to, 
+do a quick scan to make sure that the project is suitable for accepting contributions.
+- Commonly, channels are created on our Slack workspace, so please have a look there to see if the group/topic you are interested in is already there. 
+- If there is no channel, go to general or COVID-19 mailing list and raise a question on how to join that group/topic/project. Members working on that project would help you participate.
+
+## How can I fix something on the repository?
+
+- Raise mistakes, error or missing information on this repository by opening a Pull Request
+  - Submit trivial fixes (for example, a typo, a broken link or an obvious error)
+  - Start work on a contribution that was already asked for, or that you’ve already discussed, in a mailing list
+  - A pull request doesn’t have to represent finished work. It’s usually better to open a pull request early on, so others can watch or give feedback on your progress. Just mark it as a “WIP” (Work in Progress) in the subject line. You can always add more commits later.
+  - [Read details for open a Pull request](https://opensource.guide/how-to-contribute/#opening-a-pull-request)
+
+## How can I join the mailing list and participate in an ongoing conversation?
+
+The mailing list works via a Google group, so by [joining the group](https://groups.google.com/forum/#!forum/virtual-biohackathon) you will get new messages and will be allowed to post.
+
+We also have a [slack]((https://virtualbiohac-xt62674.slack.com/join/shared_invite/zt-cuur40oj-wdrVz50NocwVrH7vgKTdPg)) where channels (commonly corresponding to groups/topics/projects) are created and people can join and contribute to the discussion.
+
+## How can I offer dataset and other compute resources? What are the data protection policy?
+
+People are pointing to data or offering compute resources via email. A dedicated wiki entry will include more details on the known/available data and compute resources.
+
+## How can I use dataset and other compute resources? 
+
+**ADD DETAILS**
+
+## What data protection policies and licenses are applied?
+
+**ADD DETAILS**
```

Comment:
As different datasets, tools and so will be used, we cannot answer this question. We therefore ask you to double check for any limitations the data and tools you are using could have. A lot of free-access initiatives have put in place as a way to contribute to solutions around COVID-19. If the dataset/tool you want to use has any restrictions, please contact the responsible team/owner, chances are they can relax their regular policies/licenses. If not, please find an alternative data/tool to use.